packages/metro-babel-register/package.json

```diff
@@ -8,16 +8,16 @@
     "url": "git@github.com:facebook/metro.git"
   },
   "dependencies": {
-    "@babel/core": "7.0.0-beta.56",
-    "@babel/plugin-proposal-class-properties": "7.0.0-beta.56",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "7.0.0-beta.56",
-    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.56",
-    "@babel/plugin-proposal-optional-catch-binding": "7.0.0-beta.56",
-    "@babel/plugin-proposal-optional-chaining": "7.0.0-beta.56",
-    "@babel/plugin-transform-async-to-generator": "7.0.0-beta.56",
-    "@babel/plugin-transform-flow-strip-types": "7.0.0-beta.56",
-    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.56",
-    "@babel/register": "7.0.0-beta.56",
+    "@babel/core": "7.0.0",
```

Comment:
Since Babel 7 releases won't contain breaking change, you can/should use `^7.0.0`

Reply:
Oh yeah, sure, silly me 😣